Userland/Utilities/rev.cpp

```diff
@@ -5,23 +5,22 @@
  */
 
 #include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
 #include <errno.h>
 #include <string.h>
 #include <unistd.h>
 
-int main(int argc, char** argv)
+ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    if (pledge("stdio rpath", nullptr) < 0) {
-        perror("pledge");
-        return 1;
-    }
+    TRY(Core::System::pledge("stdio rpath"sv));
 
     Vector<StringView> paths;
     Core::ArgsParser args_parser;
 
     args_parser.set_general_help("Concatente files to stdout with each line in reverse.");
     args_parser.add_positional_argument(paths, "File path", "path", Core::ArgsParser::Required::No);
-    args_parser.parse(argc, argv);
+    args_parser.parse(arguments);
 
     Vector<FILE*> streams;
     auto num_paths = paths.size();
```

Comment:
There is another `pledge()` further down in the code that we can TRY as well.

Reply:
Thanks for noticing, fixed.